src/main/java/Duke.java

```diff
@@ -1,10 +1,315 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    /** The Logo of Duke */
+    private static final String MSG_LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /** Line to be appended before and after a message */
+    private static final String MSG_LINE = "____________________________________________________________\n";
+
+    /** Message to be printed before to greet the user */
+    private static final String[] MSG_GREET = {"Hello! I'm Duke", "What can I do for you?"};
+
+    /** Message to be printed before exiting */
+    private static final String MSG_EXIT = "Bye. Hope to see you again soon!";
+
+    /** Message to be printed after marking task as done */
+    private static final String MSG_MARK = "Nice! I've marked this task as done:";
+
+    /** Message to be printed after changing status of task to not done */
+    private static final String MSG_UNMARK = "OK, I've marked this task as not done yet:";
+
+    /** Message to be printed after adding task to list of tasks */
+    private static final String[] MSG_ADD = {"Got it. I've added this task:", "Now you have ", " tasks in the list."};
```

Comment:
Perhaps split the line into multiple lines to keep within the 120 hard limit for characters.